internal/backend/basicstation/backend.go

```diff
@@ -278,6 +292,18 @@ func (b *Backend) handleGateway(r *http.Request, c *websocket.Conn) {
 		return
 	}
 
+	if b.verifyCN && r.TLS != nil {
```

Comment:
I would add `&& len(r.TLS.PeerCertificates) > 0`, just to make sure that an empty `PeerCertificates` slice would not crash the LoRa Gateway Bridge.